Sources/MacroTesting/AssertMacro.swift

```diff
@@ -90,8 +91,11 @@ import XCTest
 ///
 /// - Parameters:
 ///   - macros: The macros to expand in the original source string. Required, either implicitly via
-///     ``withMacroTesting(isRecording:macros:operation:)-2vypn``, or explicitly via this parameter.
-///     no fix-its to apply, or if any diagnostics are unfixable, the assertion will fail.
```

Comment:
I removed this stray line:

```swift
///      no fix-its to apply, or if any diagnostics are unfixable, the assertion will fail.
```

which appears to have been orphaned at some point. I removed it for now.